docs/classad-attributes/scheduler-classad-attributes.rst

```diff
@@ -1171,25 +846,14 @@ These attributes are only reported once a full time span has accumulated.
     The published user name is actually the file transfer queue name, as
     defined by configuration variable ``TRANSFER_QUEUE_USER_EXPR``
 
-:index:`TRANSFER_QUEUE_USER_EXPR`
```

Comment:
This seems like it's in the right place-ish?  Maybe should be moved up?  Whatever's consistent with the olther places we do this.